pytorch_blade/pytorch_blade/torch-mlir/lib/Dialect/TorchConversion/Transforms/DiscConvertTorchToDiscMhlo.cpp

```diff
@@ -123,12 +129,137 @@ class ConvertOperatorOp : public OpConversionPattern<OperatorOp> {
           qmaxAttr,
           useDynamicAttr,
           roundModeAttr);
-      rewriter.replaceOp(op, {newOp});
+    } else if (name == "torch_blade.quantize") {
+      newOutput = rewriter.create<mhlo_disc::QuantizeOp>(
+          loc,
+          resultTy,
+          input,
+          scale,
+          castedZeroPoint,
+          useSymmetricAttr,
+          axisAttr,
+          qminAttr,
+          qmaxAttr,
+          useDynamicAttr,
+          roundModeAttr);
+    } else if (name == "torch_blade.dequantize") {
+      newOutput = rewriter.create<mhlo_disc::DequantizeOp>(
+          loc,
+          resultTy,
+          input,
+          scale,
+          castedZeroPoint,
+          useSymmetricAttr,
+          axisAttr,
+          useDynamicAttr,
+          roundModeAttr);
+    } else {
+      return op.emitError("Unsupported kind of torch.operator.");
+    }
+    rewriter.replaceOp(op, newOutput->getResult(0));
+    return success();
+  }
+
+  LogicalResult convertCustomCallOp(
+      OperatorOp op,
+      OpAdaptor adaptor,
+      ConversionPatternRewriter& rewriter) const {
+    auto ctx = rewriter.getContext();
+    Location loc = op.getLoc();
+    auto operands = adaptor.operands();
+    SmallVector<Type> resultTypes;
+    for (const auto& result : op.getResults()) {
+      auto torchMlirResultTy = result.getType().dyn_cast<ValueTensorType>();
```

Comment:
better to check and emit an error if the output is not a `ValueTensorType `?

Reply:
done